optimus-manager.conf

```diff
@@ -34,20 +34,20 @@ pci_reset=no
 
 # Automatically log out the current desktop session when switching GPUs.
 # This feature is currently supported for the following DE/WM :
-# KDE Plasma, GNOME, XFCE, LXDE, Deepin, i3, Openbox, AwesomeWM, bspwm
+# KDE Plasma, GNOME, XFCE, LXDE, Deepin, i3, Openbox, AwesomeWM, bspwm, dwm
 # If this option is disabled or you use a different desktop environment,
 # GPU switching only becomes effective at the next graphical session login.
 auto_logout=yes
 
-# GPU mode to use at computer startup. Possible values: nvidia, intel, hybrid, auto
+# GPU mode to use at computer startup. Possible values: nvidia, intel, amd, hybrid-intel, hybrid-amd, auto
 # "auto" is a special mode that auto-detects if the computer is running on battery
 # and selects a proper GPU mode. See the other options below.
 startup_mode=intel
 # GPU mode to select when startup_mode=auto and the computer is running on battery.
-# Possible values: nvidia, intel, hybrid
+# Possible values: nvidia, intel, amd, hybrid-intel, hybrid-amd
 startup_auto_battery_mode=intel
 # GPU mode to select when startup_mode=auto and the computer is running on external power.
-# Possible values: nvidia, intel, hybrid
+# Possible values: nvidia, intel, amd, hybrid-intel, hybrid-amd
```

Comment:
```suggestion
# GPU mode to use at computer startup. Possible values: nvidia, intel, amd, hybrid-nvidial, hybrid-amd, auto
# "auto" is a special mode that auto-detects if the computer is running on battery
# and selects a proper GPU mode. See the other options below.
startup_mode=intel
# GPU mode to select when startup_mode=auto and the computer is running on battery.
# Possible values: nvidia, intel, amd, hybrid-intel, hybrid-amd
startup_auto_battery_mode=intel
# GPU mode to select when startup_mode=auto and the computer is running on external power.
# Possible values: nvidia, intel, amd, hybrid-nvidia, hybrid-amd
```
I would assume the hybrid mode is specifying the GPU not the CPU.
Either way `hybrid-intel` assumes that optimus-manager knows which GPU to use, but I don't think this is the intent.

Reply:
I'm quite sure what you mean here?

Reply:
What I mean is the hybrid modes are a bit vague. `hybrid-amd` seems like it's for if you have 2 dGPU's, one AMD one Nvidia and you use the AMD card as primary, but what about the other way round? and what if you want primary intel but offload to amd? 
It also could cause confusion with intel-amd systems but that's lesser problem.

Reply:
Oh I see, so in the README and general Description of the Project it does state that this is for Optimus laptops and not Intel/AMD hybrid graphics. Which is Intel/NVIDIA or AMD/NVIDIA. actually now that you say that, i need to do a look over the readme as it doesn't specifically state AMD/NVIDIA just intel/NVIDIA, lol. thanks for mentioning this. :)